doc/how_to/streamlit_migration/chat.md

```diff
@@ -0,0 +1,145 @@
+# Create Chat Interfaces
```

Comment:
## Asset for upload

![streamlit-status](https://github.com/holoviz/panel/assets/42288570/992f352f-676a-4167-aad0-1fcc2745c130)

![panel-status](https://github.com/holoviz/panel/assets/42288570/dc737d33-4458-474f-9bc3-17e7bf807896)